src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -122,6 +122,62 @@ async function initializeConnection(options: CircuitStartOptions, logger: Logger
 
   Blazor._internal.forceCloseConnection = () => connection.stop();
 
+  Blazor._internal.sendJSDataStream = (data: ArrayBufferView, streamId: string, maximumIncomingBytes: number) => {
+    // Run the rest in the background, without delaying the completion of the call to sendJSDataStream
+    // otherwise we'll deadlock (.NET can't begin reading until this completes, but it won't complete
+    // because nobody's reading the pipe)
+    setTimeout(async () => {
+      const maxMillisecondsBetweenAcks = 500;
+      let numChunksUntilNextAck = 5;
+      let lastAckTime = new Date().valueOf();
+      try {
+        // Chunk Size is the max SignalR message size less 512 bytes assumed overhead.
+        const chunkSize = maximumIncomingBytes - 512;
+        if (chunkSize <= 0) {
+          throw new Error('The chunk size must be greater than 0.')
+        }
+
+        let position = 0;
+
+        // Note: The server-side `StreamBufferCapacity` option (defaults to 10) can be configured to limit how many
+        // stream items from the client (per stream) will be stored before reading any more stream items (thus applying backpressure).
```

Comment:
I think this comment must be out-of-date since `StreamBufferCapacity` doesn't appear anywhere in the rest of this PR.

Reply:
`StreamBufferCapacity` is a [SignalR option](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.signalr.huboptions.streambuffercapacity?view=aspnetcore-5.0). I just mentioned it for future reference, should we want to revisit back pressure in this area. I can remove it if you'd prefer.